site/layouts/shortcodes/card.html

```diff
@@ -37,27 +37,27 @@ <h3 class="card-title{{ if and $metadata (not $caption) }} mb-3{{ end }}">{{ if
     <p class="card-text fw-bold text-{{ if eq $background "dark" }}light{{ else }}muted{{ end }}">{{ if $rtl }}البيانات الوصفية{{ else }}Meta data{{ end }}</p>
     {{- end -}}
     {{- if $buttons -}}
-    <ul class="nav flex-row gap-3">
+    <ul class="nav flex-row gap-3 lh-1">
       <li>
-        <a href="#">
-          <svg xmlns="http://www.w3.org/2000/svg" width="1.25rem" height="1.25rem" role="img" focusable="false" aria-hidden="true">
+        <button class="btn btn-icon btn-link btn-inverse p-0 border-0">
+          <svg xmlns="http://www.w3.org/2000/svg" width="1.25rem" height="1.25rem" focusable="false" aria-hidden="true">
             <use xlink:href="#heart"/>
           </svg>
           <span class="visually-hidden">{{ if $rtl }}مفضل{{ else }}Favorite{{ end }}</span>
-        </a>
+        </button>
       </li>
       <li>
-        <a href="#">
-          <svg xmlns="http://www.w3.org/2000/svg" width="1.25rem" height="1.25rem" role="img" focusable="false" aria-hidden="true">
+        <button class="btn btn-icon btn-link btn-inverse p-0 border-0">
+          <svg xmlns="http://www.w3.org/2000/svg" width="1.25rem" height="1.25rem" focusable="false" aria-hidden="true">
             <use xlink:href="#share"/>
           </svg>
           <span class="visually-hidden">{{ if $rtl }}يشارك{{ else }}Share{{ end }}</span>
-        </a>
+        </button>
       </li>
     </ul>
     {{- end -}}
     {{- if $action -}}
-    <button class="btn btn-secondary{{ if eq $background "dark" }} btn-inverse{{ end }}" aria-label="{{ if $rtl }}وصف الزر{{ else }}Button description{{ end }}" title="{{ if $rtl }}وصف الزر{{ else }}Button description{{ end }}">{{ if $rtl }}زر{{ else }}Button{{ end }}</button>
+    <a href="#" class="btn btn-secondary{{ if eq $background "dark" }} btn-inverse{{ end }}" aria-label="{{ if $rtl }}وصف الزر{{ else }}Button description{{ end }}" title="{{ if $rtl }}وصف الزر{{ else }}Button description{{ end }}">{{ if $rtl }}زر{{ else }}Button{{ end }}</a>
```

Comment:
If we consider that most of the time we will have links instead of actions, shouldn't we use "Link description" as labels?